index.ts

```diff
@@ -0,0 +1,308 @@
+import express = require("express");
```

Comment:
the `import = require()` syntax is meant more as a compatibility to CommonJS, [see docs](https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require); why not use
```suggestion
import * as express from 'express';
```
?

Reply:
I'm afraid that I feel like I'm out of my depth here. I understand the linked section like my version ist the correct way. I explicitly changed to this syntax because of your section because I could not find any strong argument for other syntax variants.

Reply:
The latter syntax is the [recommended one](https://www.typescriptlang.org/docs/handbook/modules.html#import) since it is the native syntax for JavaScript modules introduced with ECMAScript2015.